textworld/generator/__init__.py

```diff
@@ -149,19 +149,19 @@ def make_quest(world: Union[World, State], options: Optional[GameOptions] = None
         for i in range(1, len(chain.nodes)):
             actions.append(chain.actions[i - 1])
             if chain.nodes[i].breadth != chain.nodes[i - 1].breadth:
-                event = Event(actions)
+                event = EventCondition(actions)
```

Comment:
Since `actions` is not the first parameters of the constructor anymore, you need to write `actions=actions`.